monai/transforms/post/dictionary.py

```diff
@@ -818,6 +822,56 @@ def __call__(self, data: Mapping[Hashable, NdarrayOrTensor]) -> Dict[Hashable, N
         return d
 
 
+class GetInstanceLevelSegMapd(MapTransform):
+    """Process Nuclei Prediction with XY Coordinate Map.
+
+    Args:
+        keys: keys of the corresponding items to model output.
+        hover_key: keys of the probability map from nucleus prediction branch.
+        threshold_pred: threshold the float values of prediction to int 0 or 1 with specified theashold. Defaults to 0.5.
+        threshold_overall: threshold the float values of overall gradient map to int 0 or 1 with specified theashold.
+            Defaults to 0.4.
+        min_size: objects smaller than this size are removed. Defaults to 10.
+        sigma: std. could be a single value, or `spatial_dims` number of values. Defaults to 0.4.
+        kernel_size: the size of the Sobel kernel. Defaults to 17.
+        radius: the radius of the disk-shaped footprint. Defaults to 2.
+        allow_missing_keys: don't raise exception if key is missing.
+
+    """
+
+    backend = GetInstanceLevelSegMap.backend
+
+    def __init__(
+        self,
+        keys: KeysCollection,
+        hover_key: str,
```

Comment:
Do you you think that since we accept multiple `keys`, should we accept multiple `hover_keys` too?

Reply:
Hi @drbeh, since the hover map is computed from the label, I think in most user cases, we only have one label, so we don't need to support multiple `hover_keys`. If users use a different label for segmentation, maybe they don't need to post-process it in one program, what do you think?